applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -296,3 +328,9 @@ APP_EVENT_SUBSCRIBE(MODULE, ble_peer_event);
 #if CONFIG_DESKTOP_USB_ENABLE
 APP_EVENT_SUBSCRIBE(MODULE, usb_state_event);
 #endif
+#if CONFIG_CAF_BLE_SMP_TRANSFER_EVENTS
+APP_EVENT_SUBSCRIBE(MODULE, ble_smp_transfer_event);
+#endif
+#if CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE
+APP_EVENT_SUBSCRIBE_EARLY(MODULE, config_event);
```

Comment:
Why early subscription?

Reply:
That's a good question :)

"The module must be an early subscriber to make sure it will receive the event before the configuration channel transports (:ref:`nrf_desktop_usb_state` and :ref:`nrf_desktop_hids`). "

I assume this is still valid